lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -136,6 +140,13 @@ module Cardano.Wallet.Write.Tx
     , Shelley.UTxO (..)
     , utxoFromTxOutsInRecentEra
     , utxoFromTxOuts
+    , Shelley.txins
+    , Shelley.txinLookup
+    , Shelley.txouts
+    , Shelley.balance
+    , Shelley.sumAllValue
+    , Shelley.verifyWitVKey
+    , Shelley.getScriptHash
```

Comment:
Added a few more helper functions than the one I needed to make them more visible